### ISSUE TITLE
Incorrect warning about name collisions

### ISSUE THREAD

**Issue by dhilgarth** [imjuni/ctix#132]
In my StackNetwork.ts, I have this code:

```
interface Options {
    stackName: string;
    config: NetworkConfig;
}
export { Options as StackNetworkOptions };
```

In my StackService.ts, I have this code:

```
interface Options {
    stackName: string;
    config: ServiceConfig;
}

export { Options as StackServiceOptions };
```

But I still get these warnings:
```
   WARN     StackNetwork.ts:6:1
   > P:/modular/infrastructure-services/src/constructs/docker-stack/StackNetwork.ts:6:1
   > detect same name of export statement: "Options"

   WARN     StackService.ts:7:1
   > P:/modular/infrastructure-services/src/constructs/docker-stack/StackService.ts:7:1
   > detect same name of export statement: "Options"

```

**Comment by dhilgarth** [comment]
Unfortunately, I can't provide a PR at this time.

**Comment by imjuni** [comment]
@dhilgarth This issue was fixed in `v2.4.5`, thank you!

**Comment by dhilgarth** [comment]
I can confirm it works now, thanks!
